Files/Properties.xaml.cs

```diff
@@ -40,18 +39,22 @@ private async void Properties_Loaded(object sender, Windows.UI.Xaml.RoutedEventA
             if (App.CurrentInstance.ContentPage.IsItemSelected)
             {
                 var selectedItem = App.CurrentInstance.ContentPage.SelectedItem;
-                IStorageItem selectedStorageItem;
-                try 
-                {
-                    selectedStorageItem = await StorageFolder.GetFolderFromPathAsync(selectedItem.ItemPath);
-                }
-                catch (Exception)
+                IStorageItem selectedStorageItem = null;
+
+                if (selectedItem.PrimaryItemAttribute == StorageItemTypes.File)
                 {
-                    // Not a folder, so attempt to get as StorageFile
                     selectedStorageItem = await StorageFile.GetFileFromPathAsync(selectedItem.ItemPath);
                     var hashAlgTypeName = HashAlgorithmNames.Md5;
 
-                    ItemProperties.ItemMD5Hash = await App.CurrentInstance.InteractionOperations.GetHashForFile(selectedItem, hashAlgTypeName); // get file hash
+                    // get file hash
+                    ItemProperties.ItemMD5Hash = await App.CurrentInstance.InteractionOperations.GetHashForFile(selectedItem, hashAlgTypeName);
+
+                    PropertiesMD5Hash.Visibility = Windows.UI.Xaml.Visibility.Visible;
+                    itemMD5HashValue.Visibility = Windows.UI.Xaml.Visibility.Visible;
```

Comment:
We shouldn't be referencing the xaml directly from code-behind, it would be better to add a property to a view model for this.